metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -305,8 +315,6 @@ def __init__(self, ctx: PipelineContext, config: SnowflakeV2Config):
         self.view_definitions: FileBackedDict[str] = FileBackedDict()
```

Comment:
one of the neat things about the new sql aggregator is that it internally already has a FileBackedDict for view definitions, and defers their processing until all schemas have been loaded

so it's totally safe to just call .add_view_definition immediately, and never manually track the view definitions in the source itself